src/synopsys-action/tools-parameter.ts

```diff
@@ -82,61 +82,64 @@ export class SynopsysToolsParameter {
         }
       }
     }
-    if (parseToBoolean(inputs.POLARIS_PRCOMMENT_ENABLED)) {
-      info('Polaris PR comment is enabled')
-      if (inputs.POLARIS_PARENT_BRANCH_NAME) {
-        polData.data.polaris.branch.parent.name = inputs.POLARIS_PARENT_BRANCH_NAME
-      }
-      const prCommentSeverities: string[] = []
-      const inputPrCommentSeverities = inputs.POLARIS_PRCOMMENT_SEVERITIES
-      if (inputPrCommentSeverities != null && inputPrCommentSeverities.length > 0) {
-        const severityValues = inputPrCommentSeverities.split(',')
-        for (const severity of severityValues) {
-          if (severity.trim()) {
-            prCommentSeverities.push(severity.trim())
+    if (isPullRequestEvent()) {
+      /** Set Polaris PR comment inputs in case of non PR context */
+      if (parseToBoolean(inputs.POLARIS_PRCOMMENT_ENABLED)) {
+        info('Polaris PR comment is enabled')
+        if (inputs.POLARIS_PARENT_BRANCH_NAME) {
```

Comment:
polaris parent branch name and branch name is not mandatory in action right? but if user wants to override it we allow?